master/internal/rm/kubernetesrm/spec.go

```diff
@@ -35,8 +35,13 @@ import (
 const (
 	coscheduler = "coscheduler"
 
-	gcTask  = "gc"
-	cmdTask = "cmd"
+	gcTask            = "gc"
+	cmdTask           = "cmd"
+	userLabel         = "determined_user"
+	workspaceLabel    = "determined_workspace"
+	resourcePoolLabel = "determined_resource_pool"
+	taskTypeLabel     = "determined_task_type"
+	taskIDLabel       = "determined_task_id"
```

Comment:
do we want to instead use a domain name prefix like ```determined.ai/user```? I haven't fully read the docs to see the best practices on this


https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set